### PR TITLE
input  component added

### PR DESCRIPTION
**Pull request by swstica**

### CODE REVIEW

File: input/input.css [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    box-shadow:  0 5px  5px var(--shadow-color), 0 6px 20px 0 var(--shadow-color);
+    padding: 2rem;
+}
+
+.input-box span{
+    color: red;
+}
+.input-box label{
+    padding: 2px;
+    font-size: 12px;
```
Comment (Sahil25061999) [inline input/input.css:30]: looks good, if possible use rem for values greater than 10px

Comment (swstica) [reply input/input.css:30]: Actually, 1rem was getting too big and it's said that we should prefer px over rem if a value is <1.

**Review by pradipgharge — COMMENTED** [review]
clean code, Looks good to merge!

**Review by Mohitk391 — APPROVED** [review]
nice ui..looks good to merge

**Review by manish-GH — APPROVED** [review]
As far as I know, we are supposed to use rem for quantities greater than 10px.
In your case, you could replace 12px with 0.75rem.
Just a suggestion, you could make your sidebar extend to the bottom of the screen and have its position sticky

**Pull request merged**